hacktoberfest/intros.md

```diff
@@ -3,4 +3,10 @@
 | 1 | Rahul Arulkumaran | <a href="https://github.com/rahulkumaran">rahulkumaran</a> | Hyderabad, India | Python, Full Stack, ML, Data Science |
 | 2 | Juan Luis C. | <a href="https://github.com/luizhdz">Luiz Hdz</a> | Monterrey, Mexico | Python, JS, Ruby, PHP |
 | 3 | André Monteiro | <a href="https://github.com/AndreAlbuquerque9">André Albuquerque</a> | Pernambuco, Brazil | JS, Java, C# |
-| 4 | Sean B | <a href="https://github.com/seanysean">seanysean</a> | New Brunswick, Canada | HTML, CSS |
\ No newline at end of file
+<<<<<<< HEAD
```

Comment:
This PR can't be merged. I'm guessing you'll have to make a new one.
Mainly because the conflicts haven't been resolved rightly.